app/src/main/java/pl/pola_app/helpers/Utils.java

```diff
@@ -21,8 +22,14 @@
     public static final String URL_POLA_TWITTER= "https://twitter.com/pola_app";
     public static final long TIMEOUT_SECONDS = 20;
 
-    public static String getDeviceId(Context context) {
-        return Settings.Secure.getString(context.getContentResolver(), Settings.Secure.ANDROID_ID);
+    private static String sessionGuid = null;
+
+    public static String getSessionGuid() {
+        if(sessionGuid == null) {
+            sessionGuid = UUID.randomUUID().toString();
+        }
+
+        return sessionGuid;
```

Comment:
for now guid is created once per application start. Don't we want to make it persistent?